Shared/Samples/Show utility associations/ShowUtilityAssociationsView.swift

```diff
@@ -0,0 +1,271 @@
+            } catch {
+                return
+            }
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                legend
+            }
+        }
+    }
+}
+
+private extension ShowUtilityAssociationsView {
+    /// The legend at the bottom of the screen.
+    var legend: some View {
+        HStack {
+            Label {
+                Text("Attachment")
+            } icon: {
+                if let attachmentImage {
+                    Image(uiImage: attachmentImage)
+                } else {
+                    Color.clear
+                }
+            }
+            .task(id: displayScale) {
+                attachmentImage = try? await Symbol.attachment
+                    .makeSwatch(scale: displayScale)
+            }
+            Spacer()
+            Label {
+                Text("Connectivity")
+            } icon: {
+                if let connectivityImage {
+                    Image(uiImage: connectivityImage)
+                } else {
+                    Color.clear
+                }
+            }
+            .task(id: displayScale) {
+                connectivityImage = try? await Symbol.connectivity
+                    .makeSwatch(scale: displayScale)
+            }
+        }
+        .labelStyle(.titleAndIcon)
+    }
+}
+
+private extension ShowUtilityAssociationsView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// The map with the utility network.
+        let map = Map(basemapStyle: .arcGISTopographic)
+        
+        /// The utility network for this sample.
+        private let network = UtilityNetwork(url: .utilityNetwork)
+        
+        /// A container for associations results.
+        let associationsOverlay = makeAssociationsOverlay()
+        
+        /// A Boolean value indicating if the sample is authenticated.
+        private var isAuthenticated: Bool {
+            !ArcGISEnvironment.authenticationManager.arcGISCredentialStore.credentials.isEmpty
+        }
+        
+        /// The max scale for the viewpoint.
+        private var maxScale: Double { 2_000 }
+        
+        init() {
+            map.initialViewpoint = .initialViewpoint
+            map.addUtilityNetwork(network)
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        // MARK: Methods
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        func setup() async {
+            do {
+                try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+                try await network.load()
+                addLayers()
+            } catch {
+                return
+            }
```

Comment:
Instead of discarding errors like this, it might be better to make this method throwing and then let the caller decide how they want to handle the error. If the caller wants to, they could just call this method using `try?` to discard the error if one is thrown.